src/lib/editor.ts

```diff
@@ -1,20 +1,37 @@
-export type SupportedEditor =
-	| 'vscode'
-	| 'phpstorm'
-	| 'cursor'
-	| 'windsurf'
-	| 'nova'
-	| 'webstorm'
-	| 'sublime'
-	| 'atom';
+export type SupportedEditor = 'vscode' | 'phpstorm' | 'cursor' | 'windsurf' | 'webstorm';
```

Comment:
We have decided to continue with the five listed editors only. Read more: https://linear.app/a8c/issue/STU-354/curate-list-of-code-editorsterminals-apps-to-be-supported-in-studio#comment-e2451a71